chromadb/test/db/test_system.py

```diff
@@ -360,6 +371,7 @@ def test_create_get_delete_database_and_collection(sysdb: SysDB) -> None:
         name=sample_collections[0]["name"], database="new_database"
     )
     assert len(result) == 1
+    sample_collections[0]["database"] = "new_database"
```

Comment:
this feels hacky? Why are we setting the db after the fact?

Reply:
The reason I had to set it is because in the predefined `sample_collections`, we could have only "hard-coded" values. I can use a generator for this.

Reply:
ok, can we clone it at the beginning of test? Otherwise this shared state is a bit buggy

Reply:
leaving as understandable tech debt